hs628_tools/Test.py

```diff
@@ -0,0 +1,133 @@
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+
+
+
+
+
+class data_split:
+    """
+    Create class data_split to predict a perfect train and test data split ratio.
+
+    defines method to accomplish this task.
+
+    returns a split ratio.
+
+    Identifies Missing values as Nan for any missing value, return train and test dataframe with uniform division
+    of NAN between train and test data per the data split ratio.
+
+    """
+    def __init__(self, data):
+        self.data = data
+
+
+    def test_train():
+        """
+        Reads in csv file using np.genfromtxt, counts number of NANs in the dataset
+
+        """
+
+        data = np.genfromtxt(csv_datafile, delimiter=',', skip_header=1)
+        np.isnan(data).sum() #counts the number of "NANs" in dat
```

Comment:
Hi @nirveshk, I'd like your idea of split data. And I may use your idea to read my data instead my initial way. 
Just, make sure, so for the csv_datafile here, will it be the parameter of your function?

Reply:
@EHWUSF I have hard coded my datasets while reading it with np.genfromtxt. I have also added a url link to where that data is available(in my github account under HS628 project1 repo). This way user can either download the data from the link provided or add their own data to use the python package I created. I read this suggestion in one of your response to one of our classmate's pull request and I decided to go with it. Thanks!